### DISCUSSION TITLE
Help :_(

### DISCUSSION THREAD

**Discussion by reanoff** [chr233/ASFEnhance#36] [提问求助 / Support] [answered]
>
PUBLISHRECOMMENT ASF 440
<
ASFenhance encountered error, error logs below
==========================================
 - Origin Message: PUBLISHRECOMMENT ASF 440
 - Access: Owner
 - ASF Version: 5.3.2.4
 - Plugin Version: 1.6.19.3
==========================================
null
==========================================
 - Exception Type: System.NullReferenceException
 - Exception Message: Object reference not set to an instance of an object.
   at ASFEnhance.ASFEnhance.ResponseCommand(Bot bot, EAccess access, String message, String[] args, UInt64 steamID)
   at ASFEnhance.ASFEnhance.OnBotCommand(Bot bot, EAccess access, String message, String[] args, UInt64 steamID)

**Comment by chr233** [comment]
![image](https://user-images.githubusercontent.com/13798576/203508354-c79fee2d-97d1-45ec-845c-a94ccfef6bef.png)

you missed argument "comment"

**Reply by reanoff** [reply]
>
prec asf 440 COMMENT : asddsadasd
<
ASFenhance encountered error, error logs below
==========================================
 - Origin Message: prec asf 440 COMMENT : asddsadasd
 - Access: Owner
 - ASF Version: 5.3.2.4
 - Plugin Version: 1.6.19.3
==========================================
null
==========================================
 - Exception Type: System.NullReferenceException
 - Exception Message: Object reference not set to an instance of an object.
   at ASFEnhance.ASFEnhance.ResponseCommand(Bot bot, EAccess access, String message, String[] args, UInt64 steamID)
   at ASFEnhance.ASFEnhance.OnBotCommand(Bot bot, EAccess access, String message, String[] args, UInt64 steamID)

**Comment by chr233** [comment]
I will let this command's output more friendly

**Reply by reanoff** [reply]
I wrote incorrectly?

**Reply by reanoff** [reply]
>
PUBLISHRECOMMENT ASF 440 lol
<
ASFenhance encountered error, error logs below
==========================================
 - Origin Message: PUBLISHRECOMMENT ASF 440 lol
 - Access: Owner
 - ASF Version: 5.3.2.4
 - Plugin Version: 1.6.19.3
==========================================
null
==========================================
 - Exception Type: System.NullReferenceException
 - Exception Message: Object reference not set to an instance of an object.
   at ASFEnhance.ASFEnhance.ResponseCommand(Bot bot, EAccess access, String message, String[] args, UInt64 steamID)
   at ASFEnhance.ASFEnhance.OnBotCommand(Bot bot, EAccess access, String message, String[] args, UInt64 steamID)

**Comment by chr233** [comment]
It's a bug, still working on it

**Comment by chr233** [comment, marked as answer]
try 1.6.19.4， it should be fixed

**Reply by reanoff** [reply]
The plugin is up to date, no need to update

**Reply by reanoff** [reply]
How can I update the plugin?

**Reply by chr233** [reply]
fixed. you can update to 1.6.20.1
